cmd/csilvm/csilvm.go

```diff
@@ -84,13 +95,66 @@ func main() {
 	if len(*nodeIDF) > defaultMaxStringLen {
 		log.Fatalf("node-id cannot be longer than %d bytes: %q", defaultMaxStringLen, *nodeIDF)
 	}
+	scope := tally.NoopScope
+	if *statsdUDPHostEnvVarF != "" && *statsdUDPPortEnvVarF != "" {
+		statsdHost := os.Getenv(*statsdUDPHostEnvVarF)
+		statsdPort := os.Getenv(*statsdUDPPortEnvVarF)
+		statsdServerAddr := fmt.Sprintf("%s:%s", statsdHost, statsdPort)
+		// Set no statsd prefix, tags are already prefixed using 'csilvm'.
+		const (
+			statsdPrefix     = ""
+			maxFlushInterval = time.Second
+			maxUDPPacketSize = 1440
```

Comment:
should this be configurable?

Reply:
Yeah - I think so - especially if the metrics server is not local ... in which case we will want to set this to 512. Also - I see the library prefers a slightly smaller value than 1440.